cmd/fpga_admissionwebhook/fpga_admissionwebhook.go

```diff
@@ -219,38 +214,32 @@ func main() {
 	var mode string
 	var config *rest.Config
 	var err error
-	var debugEnabled bool
 
 	flag.StringVar(&kubeconfig, "kubeconfig", "", "absolute path to the kubeconfig file")
 	flag.StringVar(&master, "master", "", "master url")
 	flag.StringVar(&certFile, "tls-cert-file", certFile,
 		"File containing the x509 Certificate for HTTPS. (CA cert, if any, concatenated after server cert).")
 	flag.StringVar(&keyFile, "tls-private-key-file", keyFile, "File containing the x509 private key matching --tls-cert-file.")
 	flag.StringVar(&mode, "mode", preprogrammed, fmt.Sprintf("webhook mode: '%s' (default) or '%s'", preprogrammed, orchestrated))
-	flag.BoolVar(&debugEnabled, "debug", false, "enable debug output")
 	flag.Parse()
 
-	if debugEnabled {
-		debug.Activate()
-	}
-
 	if certFile == "" {
-		fmt.Println("TLS certificate file is not set")
+		klog.Fatal("TLS certificate file is not set")
 		os.Exit(1)
```

Comment:
redundant `os.Exit(1)`

Reply:
ack. I did consider dropping them - was not sure how 'obvious' it needed to be in the code. `Exit()`'s dropped...